integration/dev_test.go

```diff
@@ -26,16 +26,16 @@ import (
 	"testing"
 	"time"
 
-	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/client-go/tools/clientcmd"
-
 	"github.com/GoogleContainerTools/skaffold/integration/skaffold"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/config"
 	"github.com/GoogleContainerTools/skaffold/proto"
 	"github.com/GoogleContainerTools/skaffold/testutil"
+	"github.com/sirupsen/logrus"
```

Comment:
This breaks the linter

Reply:
ah, yeah, it's time to setup in my new IDE the filewatchers for `goimports -w -local github.com/GoogleContainerTools/skaffold` :)